app/src/pages/inside/dashboardPage/widgets/uniqueBugsTable/uniqueBugsTable.stories.jsx

```diff
@@ -0,0 +1,33 @@
+import { storiesOf } from '@storybook/react';
+import { host } from 'storybook-host';
+import { withReadme } from 'storybook-readme';
+import 'c3/c3.css';
```

Comment:
It's not necessary. This import has been moved to the storybook config.

Reply:
Removed